neossat2caom2/storage_name.py

```diff
@@ -78,33 +78,19 @@ class NEOSSatName(StorageName):
 
     def __init__(self, file_name, source_names):
         super().__init__(file_name=file_name, source_names=source_names)
-        self._logger.debug(self)
-
-    def __str__(self):
-        return f'\n' \
-               f'      obs id: {self._obs_id}\n' \
-               f'   file name: {self._file_name}\n' \
-               f'source names: {self.source_names}\n'
 
     def is_valid(self):
         return True
+
     @property
     def prev(self):
         """The preview file name for the file."""
-        return '{}_{}_prev.png'.format(self.obs_id, self._product_id)
+        return f'{self._obs_id}_{self._product_id}_prev.png'
```

Comment:
Changed from `self.obs_id` to `self._obs_id`.  Error?

Reply:
The naming convention, because the language doesn't enforce public/private, is that 'private' attributes have a single underscore prefix, and the @property annotation dispenses with the underscore.

So, within the class, use the underscores, outside the class, use the attributes.

This is an old class, one of the first Python ones I wrote, and I didn't understand that distinction when I wrote it. This is me fixing my lack of understanding.

Does that make sense?

Reply:
Nope, but OK.